lib/rspec/sidekiq/matchers/be_deadable.rb

```diff
@@ -0,0 +1,41 @@
+module RSpec
+  module Sidekiq
+    module Matchers
+      def be_deadable
+        BeDeadable.new(false)
+      end
+
+       def not_be_deadable
+        BeDeadable.new(true)
+      end
```

Comment:
This is still more confusing than it needs to be. RSpec allows you to negate a matcher (i.e. `expect(myjob).not_to be_dead)` works as long as I defined `be_dead`), so we don't need to define both. That way, the class doesn't need to take an argument because it's always looking for `sidekiq_options['dead'] == true`.

Also like I said earlier, this really needs to be `BeDead` and `be_dead` for the matcher, deadable doesn't really make any sense.